darwin/torch/dataset.py

```diff
@@ -320,16 +343,16 @@ def get_target(self, index: int) -> Dict[str, Any]:
 
         return target
 
-    def measure_weights(self, **kwargs) -> np.ndarray:
+    def measure_weights(self) -> np.ndarray:
```

Comment:
Can we try and use the following?

```python
import numpy.typing as npt

...

    def measure_weights(self) -> npt.NDArray[np.float64]:
```

I'm not sure `np.float64` is the right type here, do double check

(This comment is valid for all functions where we return a `np.ndarray`)

Reply:
done